src/Evolution/Systems/ScalarWave/Equations.hpp

```diff
@@ -97,6 +98,103 @@ struct ComputeNormalDotFluxes {
                         interface_unit_normal) noexcept;
 };
 
+/*!
+ * \ingroup NumericalFluxesGroup
+ * \brief Computes the penalty flux
+ *
+ * The penalty flux is given by:
+ * \f{align}
+ * F^*(\Psi) =& F(\Psi) = 0 \\
+ * F^*(\Pi) =& F(\Pi)_{\mathrm{int}} + \\
+ *         \frac{1}{2} p \left(U^-_{\mathrm{ext}} - U^-_{\mathrm{int}}\right) \\
+ * F^*(\Phi_i) =& F(\Phi_i)_{\mathrm{int}} - \\
+ *         \frac{1}{2} p \left(U^-_{\mathrm{ext}} - U^-_{\mathrm{int}}\right)
+ * \f}
+ * where \f$F^*\f$ is the normal dotted with the numerical flux and \f$F\f$ is
+ * the normal dotted with the flux, which is computed in
+ * ScalarWave::ComputeNormalDotFluxes.
```

Comment:
Since `p` is always chosen to be one that should be stated explicitly.

Reply:
stated